src/extension.ts

```diff
@@ -1396,6 +1407,7 @@ export class Ext extends Ecs.System<ExtEvent> {
     /** Triggered when a grab operation has been started */
     on_grab_start(meta: null | Meta.Window, op: any) {
         if (!meta) return
+        this.grab_time = +new Date();  // timestamp
```

Comment:
Is this an appropriate way to get a timestamp in gjs?

I wanted to use `this.grab_time = global.get_current_time();` but it only seems to return `0`. Maybe it doesn't do what I think it does, couldn't find docs.